src/plugins/discover/public/application/components/json_code_editor/json_code_editor.scss

```diff
@@ -0,0 +1,3 @@
+.jsonCodeEditor {
```

Comment:
To be on the safe side, we should use a prefix (`.dscJsonCodeEditor`) so as to mitigate the potential for conflicting styles elsewhere in Kibana. The `dsc` prefix is common in most other Discover `scss` files.